gateway/response/response.go

```diff
@@ -75,6 +75,8 @@ const (
 
 	transPixelResponse = "\x47\x49\x46\x38\x39\x61\x01\x00\x01\x00\x80\x00\x00\x00\x00\x00\x00\x00\x00\x21\xF9\x04" +
 		"\x01\x00\x00\x00\x00\x2C\x00\x00\x00\x00\x01\x00\x01\x00\x00\x02\x02\x44\x01\x00\x3B"
+
+	InvalidMessage = "missing required fields in message"
```

Comment:
It seems to be too generic. Should we scope it down to stream message? wdyt?
```suggestion
	InvalidStreamMessage = "missing required fields in stream message"
```